great_expectations/expectations/metrics/table_metrics/table_column_types.py

```diff
@@ -106,7 +110,7 @@ def _get_sqlalchemy_column_metadata(
 
     return get_sqlalchemy_column_metadata(
         execution_engine=execution_engine,
-        table_selectable=table_selectable,  # type: ignore[arg-type]
+        table_selectable=_get_normalized_table_name(table_selectable),  # type: ignore[arg-type]
```

Comment:
See my comment above about only calling this normalization for MSSQL execution engines.